build/common.sh

```diff
@@ -40,6 +40,9 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 readonly KUBE_BUILD_IMAGE_REPO=kube-build
 readonly KUBE_BUILD_IMAGE_CROSS_TAG="$(cat "${KUBE_ROOT}/build/build-image/cross/VERSION")"
 
+readonly KUBE_DOCKER_REGISTRY="${KUBE_DOCKER_REGISTRY:-k8s.gcr.io}"
+readonly BASE_IMAGE_REGISTRY="${BASE_IMAGE_REGISTRY:-k8s.gcr.io}"
```

Comment:
I think prefixing this with `KUBE_` better follows convention

Reply:
@fejta make sense, i can fix it